### PR TITLE
K8SPXC-917  Update for PXC operator release 1.11.0

### PR DESCRIPTION
**Pull request by nmarukovich**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.11.0.pxc-operator.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "c3991f0959a3b4114d7ff629d9d3cdf0dc200c58443ca8ebb1446d8b1cbe416d",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.23": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc8.0-backup",
+            "image_hash": "6ab8efb3804d1e519e49ee10eb46b428a837cfdcee222cc5ae2089cc1dc02a6d",
+            "status": "available",
+            "critical": false
+          },
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:195]: I think these need to be removed. Only one backup image should exists for major version unless we add more after the release.

File: sources/operator.1.11.0.pxc-operator.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "ac9fcd3078107c6492c687eb98215d4e5daf27a02fb3c78ba4b9e9c01f2078b3",
+            "status": "available",
+            "critical": false
+          },
+          "2.4.24": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup",
+            "image_hash": "2ff5992220ba251cf064cc2b4d5929e0fdb963db18e35d6c672f9aacb0be3bed",
+            "status": "available",
+            "critical": false
+          }
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:213]: Need to be removed.

File: sources/operator.1.11.0.pxc-operator.json [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "6ab8efb3804d1e519e49ee10eb46b428a837cfdcee222cc5ae2089cc1dc02a6d",
+            "status": "available",
+            "critical": false
+          },
+          "2.4.26-1": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup",
+            "image_hash": "0424bb866cb171bd380c87ad3aa98047af90c1e6e61bf4d81d2c36864cb36e4d",
+            "status": "recommended",
+            "critical": false
+          },
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:201]: After removal of below lines comma will need to be removed.

File: sources/operator.1.11.0.pxc-operator.json [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "d69dad98900532e2ad6d0bf12c34a148462816fa3ee4697e9b73efef7583901a",
+            "status": "available",
+            "critical": false
+          },
+          "1.10.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-1-logcollector",
+            "image_hash": "8f106b1e9134812b77f4e210ad0fcd7d8d3515a90fe53554d24cd49defc9e044",
+            "status": "available",
+            "critical": false
+          }
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:233]: Need to be removed. Only one should exist.

File: sources/operator.1.11.0.pxc-operator.json [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+          },
+          "2.4.24": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup",
+            "image_hash": "2ff5992220ba251cf064cc2b4d5929e0fdb963db18e35d6c672f9aacb0be3bed",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "log_collector": {
+          "1.8.11": {
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:216]: Do we want to start putting fluentbit version here? @hors

File: sources/operator.1.11.0.pxc-operator.json [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "b84701c47a11c6f5ca46481f25f1b6086c0a30014d05584c7987f1d42a17b584",
+            "status": "available",
+            "critical": false
+          },
+          "2.0.18": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-proxysql",
+            "image_hash": "f109a62eb316732d59dd80ed0e013fc9594cbae601586b94023b8c068f7ced7b",
+            "status": "available",
+            "critical": false
+          }
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:155]: Only one image should exist here so last two need to be removed.

File: sources/operator.1.11.0.pxc-operator.json [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+            "image_hash": "62479be2a21192a3215f03d3f9541decd5ef1737741245ac33ee439915a15128",
+            "status": "available",
+            "critical": false
+          },
+          "2.3.14": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-haproxy",
+            "image_hash": "2f06ac4a0f39b2c0253421c3d024291d5ba19d41e35e633ff6ddcf4ba67fd51a",
+            "status": "available",
+            "critical": false
+          }
```
Comment (tplavcic) [inline sources/operator.1.11.0.pxc-operator.json:175]: Only one image should exist here so last two need to be removed.

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.11.0.pxc-operator.json (hunk 5 above)
Comment (hors) [reply sources/operator.1.11.0.pxc-operator.json:168]: lets use old approach (use operator version)

**Review by hors — CHANGES_REQUESTED** [review]

File: sources/operator.1.11.0.pxc-operator.json (hunk 5 above)
Comment (nmarukovich) [reply sources/operator.1.11.0.pxc-operator.json:168]: ok) it didn't catch it)

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Pull request merged**